util/randomutil/randomutil_test.go

```diff
@@ -0,0 +1,14 @@
+package randomutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGenerateInt63(t *testing.T) {
+	randomNumberGenerator := RandomNumberGenerator{}
+	num1 := randomNumberGenerator.GenerateInt63()
+	num2 := randomNumberGenerator.GenerateInt63()
+	assert.Equal(t, false, num1 == num2)
+}
```

Comment:
You can remove this test. We don't need to test the randomness of the random number generator. Statistically, this test will sometimes fail.